modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1000,11 +1002,21 @@ inline void fill_codec_context(AVCodecContext * enc, AVDictionary * dict)
     }
 }
 
+static bool isThreadSafe() {
+    const bool threadSafe = utils::getConfigurationParameterBool("OPENCV_FFMPEG_IS_THREAD_SAFE", false);
+    if(threadSafe)
```

Comment:
I think it is better to add `{` and `}` to this `if`, because `CV_LOG_WARNING` is a macro and can expand to some weird or invalid construction.

Currently only INFO, DEBUG and VERBOSE might be stripped: 
https://github.com/opencv/opencv/blob/001a2c51958ebe332562df13e9630768b66def57/modules/core/include/opencv2/core/utils/logger.hpp#L136-L149
but WARNING removal is possible in future or a custom build.